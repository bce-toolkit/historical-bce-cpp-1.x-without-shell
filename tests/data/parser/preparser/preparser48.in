S8+Ca(OH)2=CaS5+CaS2O3+H2O