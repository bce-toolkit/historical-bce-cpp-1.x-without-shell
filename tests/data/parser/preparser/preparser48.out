S8 0 0
Ca(OH)2 0 0
CaS5 1 0
CaS2O3 1 0
H2O 1 0
